pkg/operator/cluster/ceph/osd/osd.go

```diff
@@ -168,9 +169,16 @@ func (c *Cluster) Start() error {
 			errorMessages = append(errorMessages, fmt.Sprintf("failed to set orchestration starting status for node %s: %+v", n.Name, err))
 			continue
 		}
-
+		devicesToUse := n.Devices
```

Comment:
@ganeshmaharaj here you go

Reply:
`n.Devices` is the set of desired devices, right?

Reply:
yes